USER_GUIDE.md

```diff
@@ -153,6 +153,7 @@ print(response)
 - [Search](guides/search.md)
 - [Point in Time](guides/point_in_time.md)
 - [Using a Proxy](guides/proxy.md)
+- [Taking a Snapshot](guides/snapshot.md)
```

Comment:
This doc is more than taking, let's say "Working with Snapshots" or "Snapshot Actions" to match what's in the doc?